docs/pages/build/building-from-github.mdx

```diff
@@ -214,10 +214,41 @@ On your project's GitHub settings page in the Expo dashboard, you can click the
 a build trigger row to disable, edit, or delete the trigger.
 
 <ImageSpotlight
-  alt="Manage GitHub build trigger with the options button"
-  src="/static/images/eas-build/build-from-github/managing-build-triggers.png"
+  alt="The options button on the build trigger row"
+  src="/static/images/eas-build/build-from-github/edit-trigger-option.png"
 />
 
+You can also run a GitHub build with the parameters from the trigger manually. This will not count towards your automatic build trigger record.
+
+#### Automatic Store Submission with EAS Submit
+
+Once your build completes, you can automatically submit your app to the digital stores using EAS Submit. This feature streamlines the process, reducing the manual steps required to publish your app.
+
+To enable automatic submission, you need to configure your build triggers to include submission as part of the build process. Here's how you can set it up:
+
+- Navigate to your project's GitHub settings page on the Expo dashboard.
+- Find the build trigger you want to modify, and click the options button.
+- Select "Edit trigger" and in the dialog that appears, check the option "Submit to store after
+  build".
+
+<ImageSpotlight
+  alt="The EAS Submit form fields on the trigger edit form"
+  src="/static/images/eas-build/build-from-github/eas-submit-form-fields.png"
+/>
+
+- Save your changes.
+
+<ImageSpotlight
+  alt="An enabled build trigger with automatic submission enabled"
+  src="/static/images/eas-build/build-from-github/auto-submit-trigger-column.png"
+/>
```

Comment:
```suggestion
<ImageSpotlight
  alt="The EAS Submit form fields on the trigger edit form"
  src="/static/images/eas-build/build-from-github/eas-submit-form-fields.png"
  style={{ maxWidth: 640 }}
/>

- Save your changes.

<ImageSpotlight
  alt="An enabled build trigger with automatic submission enabled"
  src="/static/images/eas-build/build-from-github/auto-submit-trigger-column.png"
  style={{ maxWidth: 400 }}
/>
```

Let's also add width to the ImageSpotlight components here. Output:

![CleanShot 2024-04-30 at 19 12 47@2x](https://github.com/expo/expo/assets/10234615/4d542b9d-5908-432a-9f5e-bc38f3659e49)